### ISSUE TITLE
Could DivideMix generalize to Segmentation Problem?

### ISSUE THREAD

**Issue by riven314** [LiJunnan1992/DivideMix#5]
Thanks for the amazing work! 
I see your work mainly use object classification problem as benchmarks (as most research works of similar area do), do you think the framework could be applied on segmentation problem as well?

**Comment by LiJunnan1992** [comment]
Thanks for your interest!
Yes I think the framework can apply to object detection/segmentation. Similar as classification, the confidence of a ground-truth mask/box can be measured by its loss, and the training data can be divided into a labeled set and an unlabeled set. It may not be straightforward to apply mix-match to segmentation problems, but I'm sure there are many semi-supervised segmentation methods that can be used.

**Comment by Jo-wang** [comment]
> Thanks for the amazing work! I see your work mainly use object classification problem as benchmarks (as most research works of similar area do), do you think the framework could be applied on segmentation problem as well?

Hi,
Have you try DivideMix on segmentation task? Will it have a better performance? Thank you!
